airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -225,6 +225,7 @@ class KubernetesPodOperator(BaseOperator):
         container name to use.
     :param deferrable: Run operator in the deferrable mode.
     :param poll_interval: Polling period in seconds to check for the status. Used only in deferrable mode.
+    :param log_pod_on_failure: If pod template should be logged in a case of task fail.
```

Comment:
```suggestion
    :param log_pod_spec_on_failure: If pod template should be logged in a case of task fail.
```
or pod_specification perhaps.

Reply:
agree, renamed.